tar: add brackets explaining mnemonics (#5083)
tldr-pages/tldr: pages/common/tar.md

> Often combined with a compression method, such as gzip or bzip.
> More information: <https://www.gnu.org/software/tar>.

> - [C]reate an archive from [f]iles:

Reviewer: Maybe we should use a lower case `C` because the options are case sensitive:
```suggestion
- [c]reate an archive from [f]iles:
```
Reference: https://github.com/tldr-pages/tldr-lint/pull/12#discussion_r481134812

Author: Done.

`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`

- [C]reate a g[z]ipped archive from [f]iles:

`tar czf {{target.tar.gz}} {{file1}} {{file2}} {{file3}}`

- [C]reate a g[z]ipped archive from a directory using relative paths:

`tar czf {{target.tar.gz}} -C {{path/to/directory}} .`

- E[x]tract a (compressed) archive [f]ile into the current directory:

`tar xf {{source.tar[.gz|.bz2|.xz]}}`

> - [E]xtract a (compressed) archive [f]ile into the target directory:

Reviewer: ```suggestion
- E[x]tract a (compressed) archive [f]ile into the target directory:
```

Author: Oops!!!!!

> `tar xf {{source.tar[.gz|.bz2|.xz]}} -C {{directory}}`

Reviewer: We can use the long format of `-C` to avoid explain that letter:
```suggestion
`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{directory}}`
```

Author: Good idea.
